### PR TITLE
Replace "Final Call" by "Call for Review" in PR template and Process document

### PR DESCRIPTION
**Pull request by Jym77**
Since we now call them "Call for Review" 😃 

Closes issue(s):

- N/A

Need for Final Call:
This will not require a Final Call (change to process documents)

---

### **When creating PR:**

- [X] Make sure you're requesting to **pull a branch** (right side) to the `develop` branch (left side).
- [X] Make sure you do not remove the "How to Review and Approve" section in your pull request description

### **After creating PR:**

- [X] Add yourself (and co-authors) as "Assignees" for PR.
- [X] Add label to indicate if it's a `Rule`, `Definition` or `Chore`.
- [ ] [Link the PR to any issue it solves](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue). This will be done automatically by [referencing the issue](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) at the top of this comment in the indicated place.
- [X] Optionally request feedback from anyone in particular by assigning them as "Reviewers".

### **When merging a PR:**

- [ ] Close any issue that the PR resolves. This will happen automatically upon merging if the PR was correctly linked to the issue, e.g. by referencing the issue at the top of this comment.

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Final Call period. In case of disagreement, the longer period wins.

### CODE REVIEW

**Comment by Jym77** [comment]
@WilcoFiers #461 should probably be updated to replace "See all Pull Requests marked as Final Call here" by "See all Pull Requests marked as **Call for Review** here"

**Review by WilcoFiers — APPROVED** [review]

File: pages/design/process.md [hunk 1]
```diff
@@ -133,71 +133,73 @@ Rules that have been fixed up by the author and are ready for review again, aimi
...
-These changes are considered non-substantial and will not require a "Final call" before being published:
+These changes are considered non-substantial and will not require a "Call for Review" before being published:
 
 - Editorial changes, that do not change the meaning of a rule (also if it's in the Applicability, Expectations or Examples)
 - Changes to the Assumptions, Background, Accessibility Support. If changes to these sections seem to impact the possible outcomes of a rule, probably these sections have been misused.
 
 ### Passes this stage when:
 
-- No changes or only changes that _do not_ require a "Final call" is made to the rule after "Final Call" is launched.
+- No changes or only changes that _do not_ require a "Call for Review" is made to the rule after "Call for Review" is launched.
```
Comment (daniel-montalvo) [inline pages/design/process.md:169]:
```suggestion
- No changes or only changes that _do not_ require a "Call for Review" are made to the rule after "Call for Review" is launched.
```

File: pages/design/process.md [hunk 2]
```diff
@@ -133,71 +133,73 @@ Rules that have been fixed up by the author and are ready for review again, aimi
...
 
 ### Fails this stage and moves back two stages to "In Progress" when:
 
-- Changes that do require a final call is made to the rule after Final Call is launched.
+- Changes that do require a "Call for Review" is made to the rule after "Call for Review" is launched.
```
Comment (daniel-montalvo) [inline pages/design/process.md:173]:
```suggestion
- Changes that do require a "Call for Review" are made to the rule after "Call for Review" is launched.
```

File: pages/design/process.md [hunk 3]
```diff
@@ -133,71 +133,73 @@ Rules that have been fixed up by the author and are ready for review again, aimi
...
-- Follow up on feedback during the Final Call, and handle requested changes, evaluating whether they are of a type that MUST spawn a new "Final call":
-  - For changes that _does not_ require a "Final call" (see above): Implement changes as soon as possible, and dismiss outdated reviews (to let all reviewers know that the pull request is review ready), and request a new review from that person.
-  - For changes that _does_ require a new "Final call": Evaluate whether "Final call" should be allowed to run out before changes are made, or if changes should be made right away, but with a note that a new "Final call" will be required.
+- Send email out to all of ACT Rules Community Group that rule is in "Call for Review" for the next 2 weeks.
+- Follow up on feedback during the "Call for Review", and handle requested changes, evaluating whether they are of a type that MUST spawn a new "Call for Review":
+  - For changes that _does not_ require a "Call for Review" (see above): Implement changes as soon as possible, and dismiss outdated reviews (to let all reviewers know that the pull request is review ready), and request a new review from that person.
```
Comment (daniel-montalvo) [inline pages/design/process.md:183]:
```suggestion
  - For changes that _do not_ require a "Call for Review" (see above): Implement changes as soon as possible, and dismiss outdated reviews (to let all reviewers know that the pull request is review ready), and request a new review from that person.
```

File: pages/design/process.md [hunk 4]
```diff
@@ -133,71 +133,73 @@ Rules that have been fixed up by the author and are ready for review again, aimi
...
+  - For changes that _does_ require a new "Call for Review": Evaluate whether "Call for Review" should be allowed to run out before changes are made, or if changes should be made right away, but with a note that a new "Call for Review" will be required.
```
Comment (daniel-montalvo) [inline pages/design/process.md:184]:
```suggestion
  - For changes that _do_ require a new "Call for Review": Evaluate whether "Call for Review" should be allowed to run out before changes are made, or if changes should be made right away, but with a note that a new "Call for Review" will be required.
```

File: pages/design/process.md [hunk 5]
```diff
@@ -133,71 +133,73 @@ Rules that have been fixed up by the author and are ready for review again, aimi
...
 
 #### Reviewer's job for this stage:
 
 - Thorough reviews, take time to dive into the rule.
-- When commenting, please note whether you consider the suggested changes to be substantial or not (whether the change falls into the category of changes that do require a final call, or the ones the do not require a final call).
+- When commenting, please note whether you consider the suggested changes to be substantial or not (whether the change falls into the category of changes that do require a "Call for Review", or the ones the do not require a "Call for Review").
```
Comment (daniel-montalvo) [inline pages/design/process.md:189]:
```suggestion
- When commenting, please note whether you consider the suggested changes to be substantial or not (whether the change falls into the category of changes that do require a "Call for Review", or the ones that do not require a "Call for Review").
```

**Review by daniel-montalvo — CHANGES_REQUESTED** [review]

**Pull request merged**